### PR TITLE
Merge install-web steps to guide

### PR DESCRIPTION
**Pull request by jburel**
Add cors setup to the installation guides

cc @joshmoore @will-moore

### CODE REVIEW

File: ansible/templates/common/cors_configuration.sh.j2 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+#!/bin/bash
+
+set -e -u -x
+
+export OMERODIR=$OMERODIR
+export PATH={{ virtualenv_path }}/bin:$PATH
+{% endif %}
+
+omero config append omero.web.middleware '{"index": 0.5, "class": "corsheaders.middleware.CorsMiddleware"}'
```
Comment (joshmoore) [inline ansible/templates/common/cors_configuration.sh.j2:13]: Doesn't look like this section is rendering properly:
Screen Shot 2020-06-03 at 12 06 05

**Comment by joshmoore** [comment]
Also, your plan would be to remove the entire `sysadmins/unix/install-web.html?highlight=cors#setting-up-cors` section?

File: ansible/templates/common/cors_configuration.sh.j2 (hunk 1 above)
Comment (jburel) [reply ansible/templates/common/cors_configuration.sh.j2:13]: not great!!

**Comment by jburel** [comment]
@joshmoore yes I will remove the section mentioned above when this is ready

File: ansible/templates/common/cors_configuration.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% if doc %}
+Configure CORS. An ``index`` to specify the ordering of middleware classes is used. It is important to add the ``CorsMiddleware`` as the first class and ``CorsPostCsrfMiddleware`` as the last. You can specify the origin allowed to access or specify all the origin, for example:
```
Comment (will-moore) [inline ansible/templates/common/cors_configuration.sh.j2:2]: Maybe "An index is used to specify the ordering of middleware classes."
And "You can specify allowed origins in a whitelist, or allow all, for example:"

File: ansible/templates/common/install_cors.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% if doc %}
+Cross Origin Resourse Sharing allows web applications hosted at other origins to access resources from your OMERO.web installation. This can be achieved using the `django-cors-headers <https://github.com/adamchainz/django-cors-headers>`_ app with additional configuration of OMERO.web. See the `django-cors-headers <https://github.com/adamchainz/django-cors-headers>`_ page for more details on the settings.
```
Comment (will-moore) [inline ansible/templates/common/install_cors.sh.j2:2]: typo: Resource

File: ansible/templates/common/cors_configuration.sh.j2 (hunk 1 above)
Comment (jburel) [reply ansible/templates/common/cors_configuration.sh.j2:13]: fixed

**Comment by jburel** [comment]
@joshmoore @will-moore I have now fixed the typo and display issues noticed.

**Comment by will-moore** [comment]
@jburel How do you want this tested? Building the docs (to check they look OK)? How do you do that? What about functional testing? Building and Running as described in the README?

**Comment by jburel** [comment]
The installation is tested via travis. All the commands work.
You could start with a "clean" C7 docker image and follow the walkthrough if you want to try it out.

File: ansible/templates/common/cors_configuration.sh.j2 (hunk 2 above)
Comment (will-moore) [reply ansible/templates/common/cors_configuration.sh.j2]: I think the last sentence still needs fixing "You can specify the origin allowed to access or specify all the origin, for example" -> "You can specify allowed origins in a whitelist, or allow all, for example"

**Comment by will-moore** [comment]
One more fix missed "You can specify allowed origins in a whitelist, or allow all, for example:"

**Comment by will-moore** [comment]
Great, thanks 👍

**Comment by jburel** [comment]
Doc has been approved see https://github.com/ome/ome-documentation/pull/2130. Merging

**Pull request merged**